src/main/java/org/apache/maven/plugins/shade/DefaultShader.java

```diff
@@ -338,11 +342,25 @@ private void logSummaryOfDuplicates( Multimap<Collection<File>, String> overlapp
             final Collection<String> overlaps = new ArrayList<>();
             if ( !classes.isEmpty() )
             {
-                overlaps.add( "classes" );
+                if ( resources.size() == 1 )
+                {
+                    overlaps.add( "classe" );
```

Comment:
redundant e?

Reply:
I'd say french typo  ;). Thanks for the catch, should be fixed now

Reply:
I assumed so...